JetStreamDriver.js

```diff
@@ -791,8 +808,12 @@ class Benchmark {
                 addScript(text);
         } else {
             const cache = JetStream.blobDataCache;
-            for (const file of this.plan.files)
-                addScriptWithURL(cache[file].blobURL);
+            for (const file of this.plan.files) {
+                if (globalThis.prefetchResources)
+                    addScriptWithURL(cache[file].blobURL);
+                else
+                    addScriptWithURL(file);
```

Comment:
Nit: I would just make this `addScriptWithURL(globalThis.prefetchResources ? cache[file].blobURL : file);`

Reply:
done